eval.py

```diff
@@ -2,47 +2,108 @@
 import fire
 import numpy as np
 import yaml
-from selection.serialization import deserialize
+from pathlib import Path
+import pandas as pd
 
+#Constants
+label_idx = 0 #IDs look like "en/clips/and/common_voice..." this is the index of the label
 
-def main(eval_file, input_samples, train_file="train.yaml"):
 
-    eval_data = deserialize.deserialize_from_pb(eval_file)
-    eval_target = eval_data["target_mswc_vectors"]
-    eval_nontarget = eval_data["nontarget_mswc_vectors"]
-    eval_x = np.vstack([eval_target, eval_nontarget])
-    eval_y = np.concatenate(
-        [np.ones(eval_target.shape[0]), np.zeros(eval_nontarget.shape[0])]
-    )
+def create_train_sets(selected_data, allowed_training_embeddings, training_classes, train_dataset):
+    train_x = [] #embeddings
+    train_y = [] #numerical index of training_classes 
+    for selected_id in selected_data["nontargets"]:
+        if not selected_id in allowed_training_embeddings['nontargets']:
```

Comment:
```suggestion
    train_y = [] #numerical index of training_classes 
    allowed_nontargets = set(allowed_training_embeddings['nontargets'])
    for selected_id in selected_data["nontargets"]:
        if not selected_id in allowed_nontargets:
```

alternatively we could setify it in the yaml loader